app/src/main/java/org/mozilla/fenix/components/metrics/AdjustMetricsService.kt

```diff
@@ -70,12 +80,32 @@ class AdjustMetricsService(private val application: Application) : MetricsServic
         Adjust.gdprForgetMe(application.applicationContext)
     }
 
-    // We're not currently sending events directly to Adjust
-    override fun track(event: Event) { /* noop */ }
-    override fun shouldTrack(event: Event): Boolean = false
+    override fun track(event: Event) {
+        CoroutineScope(dispatcher).launch {
+            if (event is Event.GrowthData && storage.shouldTrack(event)) {
+                Adjust.trackEvent(AdjustEvent(event.tokenName))
+                storage.updateSentState(event)
+            }
+        }
+    }
+
+    override fun shouldTrack(event: Event): Boolean =
+        event is Event.GrowthData
 
     companion object {
         private const val LOGTAG = "AdjustMetricsService"
+        private const val dayMillis: Long = 1000 * 60 * 60 * 24
+        private const val windowStartMillis: Long = 2
+        private const val windowEndMillis: Long = dayMillis * 28
+
+        fun installDateWithinWindow(context: Context): Boolean {
+            val installedTime = context.packageManager
+                .getPackageInfo(context.packageName, 0)
+                .firstInstallTime
+            val timeDifference = System.currentTimeMillis() - installedTime
+
+            return timeDifference in windowStartMillis..  windowEndMillis
```

Comment:
This is defined outside of the `Storage` so that it can be mocked during tests without relying on `Context` or specific results from `System.currentTimeMillis`. Open to moving it elsewhere though